required/examples/basics/video.js

```diff
@@ -1,26 +1,13 @@
-var renderer = PIXI.autoDetectRenderer(800, 600, { transparent: true });
-document.body.appendChild(renderer.view);
-
-// create the root of the scene graph
-var stage = new PIXI.Container();
+var app = new PIXI.Application(800, 600, { transparent: true });
+document.body.appendChild(app.view);
 
 // create a video texture from a path
 var texture = PIXI.Texture.fromVideo('required/assets/testVideo.mp4');
```

Comment:
Instead of the texture automatically being created and added to the stage, on initial load this would not be the case; there would just be a 'play' sprite. When this is pressed, it would disappear, and then create the video texture, adding to a sprite, adding to the stage etc.